tests/Builder/AbstractFormContractorTest.php

```diff
@@ -97,6 +111,7 @@ public function testDefaultOptionsForSonataFormTypes(): void
             ModelListType::class,
             ModelHiddenType::class,
             ModelAutocompleteType::class,
+            'MyCustomType',
```

Comment:
Easiest way to solve this, would be to create a fixture `MyCustomType` class.

Reply:
Maybe `stdClass::class` would also work

Reply:
but cleaner and easier to understand is indeed a `MyCustomType` fixtures class